tests/python/test_clang.py

```diff
@@ -485,5 +484,25 @@ def test_printk_2s(self):
         self.assertIn(expectedWarn, output)
         r.close()
 
+    def test_map_insert(self):
+        text = """
+BPF_HASH(dummy);
+void do_trace(struct pt_regs *ctx) {
+    u64 key = 0, val = 2;
+    dummy.insert(&key, &val);
+    key = 1;
+    dummy.update(&key, &val);
+}
+"""
+        b = BPF(text=text)
+        c_val = ct.c_ulong(1)
+        b["dummy"][ct.c_ulong(0)] = c_val
+        b["dummy"][ct.c_ulong(1)] = c_val
+        b.attach_kprobe(event="sys_sync", fn_name="do_trace")
+        libc = ct.CDLL("libc.so.6")
+        libc.sync()
```

Comment:
I guess you could trace some cheaper syscall, like getuid or getpid, but I don't think it matters very much.